main.go

```diff
@@ -95,6 +98,49 @@ func init() {
 	//+kubebuilder:scaffold:scheme
 }
 
+func SetupLogging(opts *zap.Options) {
+	cfg, err := config.GetConfig()
+	if err != nil {
+		log.Error(err, "Error getting config for APIServer")
+		os.Exit(1)
+	}
+
+	clnt, err := client.New(cfg, client.Options{})
+	if err != nil {
+		log.Error(err, "can't create client from config")
+		os.Exit(1)
+	}
+	oprNamespace, err := sdkk8sutil.GetOperatorNamespace()
+	if err != nil {
+		log.Error(err, "failed to get operator namespace")
+		os.Exit(1)
+	}
+
+	oprCfg, _ := configmaps.RetrieveConfigMap(oprNamespace, "artemiscloud-operator-config", clnt)
```

Comment:
the operator deployment name is `activemq-artemis-controller-manager` maybe the configmap name could be `activemq-artemis-controller-config` or `activemq-artemis-operator-config`

Reply:
+1 for activemq-artemis-operator-config